### ISSUE TITLE
Is this project still alive?

### ISSUE THREAD

**Issue by ybtsdst** [purecloudlabs/elasticsearch-lambda#32]
In es 5.0 and later versions, the snapshot directory structure has changed.
I'm working to upgrade to 5.4.

Is this project still under development and will add more features?

**Comment by drewdahlke** [comment]
Indeed, several things have changed with ES 5.x. We're working on upgrading and there's a 5.1.1 dev branch, but at the time of writing I think we're probably a couple months out on getting that stable.
